reana_client/api/client.py

```diff
@@ -1094,7 +1145,13 @@ def get_workflow_retention_rules(workflow, access_token):
 
 
 def prune_workspace(workflow, include_inputs, include_outputs, access_token):
-    """Prune workspace files."""
+    """Prune workspace files.
+
+    :param workflow: name or id of the workflow.
+    :param include_inputs: whether to also delete inputs.
+    :param include_outputs: whether to also delete outputs.
```

Comment:
We should decide between `whether ...` or `if ``True``, ...` and use that everywhere to describe boolean parameters